src/System.Runtime.Extensions/src/System.Runtime.Extensions.builds

```diff
@@ -9,14 +9,6 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Runtime.Extensions.csproj">
-      <TargetGroup>netstandard1.5</TargetGroup>
-      <OSGroup>Unix</OSGroup>
-    </Project>
-    <Project Include="System.Runtime.Extensions.csproj">
-      <TargetGroup>netstandard1.5</TargetGroup>
-      <OSGroup>Windows_NT</OSGroup>
-    </Project>
-    <Project Include="System.Runtime.Extensions.csproj">
       <TargetGroup>net462</TargetGroup>
```

Comment:
When I remove net462 config, I am running into packaging errors that I do not know how to fix:

```
packageIndex.json is has an assembly version entry(s) for 4.1.1.0 which are no longer in package System.Runtime.Extensions.  Please run /t:UpdatePackageIndex on this project to commit an update.
```

Running `/t:UpdatePackageIndex` does nothing to the `packageIndex.json`.